src/config.rs

```diff
@@ -239,3 +245,18 @@ impl Secret {
         .with_context(|| format!("Error while decrypting secret named {}", key))
     }
 }
+
+fn find_amber_yaml<P: AsRef<Path>>(name: P) -> Result<PathBuf> {
+    if name.as_ref().is_file() {
+        return Ok(name.as_ref().to_path_buf());
+    }
+    let path = std::env::current_dir()?;
+    for file in path.ancestors() {
+        let amber_yaml: PathBuf = file.join(&name);
```

Comment:
Technically, from context, this code is fine, since `name` will always been `amber.yaml`. However, if this function was called with a `Path` that includes a directory, I think the behavior would be surprising. Instead of taking an argument here for the name, I think using the `const` value for the default file name would be better.